constants_test.py

```diff
@@ -24,9 +24,15 @@ class ConstantsTests(test_utils.GenericTestBase):
     def test_constants_file_is_existing(self):
         """Test if the constants file is existing."""
         self.assertTrue(os.path.isfile(os.path.join(
-            'assets', 'constants.json')))
+            'assets', 'constants.js')))
 
-    def test_constants_file_is_valid_json(self):
+    def test_constants_file_contains_valid_json(self):
         """Test if the constants file is valid json file."""
-        with open(os.path.join('assets', 'constants.json'), 'r') as f:
-            self.assertTrue(isinstance(json.load(f), dict))
+        with open(os.path.join('assets', 'constants.js'), 'r') as f:
+            text = f.read()
```

Comment:
Instead of replicating the logic in constants.py here, perhaps instead you should refactor the logic in constants.py into its own function, call that function, and verify that the result is a dict and contains some particular key/value pair.